apps/meteor/app/livechat/server/settingsRegenerator.ts

```diff
@@ -0,0 +1,28 @@
+// Validates settings on DB are correct on structure
+// And deletes invalid ones
+import { Settings } from '@rocket.chat/models';
+
+import { Logger } from '../../../server/lib/logger/Logger';
+
+// Validates settings on DB are correct on structure by matching the ones missing all the required fields
+const logger = new Logger('SettingsRegenerator');
+export async function settingsRegenerator() {
```

Comment:
as previously discussed as well, it would be good to move this function to a "global" namespace